integration/okhttp/build.gradle

```diff
@@ -5,7 +5,7 @@ dependencies {
     annotationProcessor project(':annotation:compiler')
 
     compile "com.squareup.okhttp:okhttp:2.7.1"
-    compile "com.android.support:support-annotations:{$ANDROID_SUPPORT_VERSION}"
```

Comment:
Huh, I'm surprised this even worked as it was before...

Reply:
Probably Gradle is lenient enough to ignore the `{}` in the version number, because `$ANDROID_SUPPORT_VERSION` was a correct way to reference the version.